### PR TITLE
Release Notes entry 2024.02.0

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** Adding Release notes for 2024.02.0
- **Is this PR related to a Monolith release?** yes 2024.02.0

### Resolves:

* [OKTA-691309](https://oktainc.atlassian.net/browse/OKTA-691309)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/5eb646e35721d1f428ae338758e32eba9978199c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2024-okta-identity-engine%2Findex.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9ee8eca1-01af-4225-99ac-c1e60395fdc6) | 61  | 56  | 64  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/release-notes/2024/index.md](https://github.com/okta/okta-developer-docs/blob/5eb646e35721d1f428ae338758e32eba9978199c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2024%2Findex.md) | [74](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7298b763-8266-4776-94f4-0c8d1d635217) | 62  | 53  | 63  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.210a05bb-c4b0-495b-9ca9-5ee1e920c6f8?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md [hunk 1]
```diff
@@ -6,6 +6,72 @@ title: Okta Identity Engine API Products release notes 2024
...
 
+## February
+
+### Monthly release 2024.02.0
+
+| Change | Expected in Preview Orgs |
+|--------|--------------------------|
+| [Assign admin roles to an app](#assign-admin-roles-to-an-app)| June 14, 2023 |
+| [DPoP support for Okta management APIs is GA in Production](#dpop-support-for-okta-management-apis-is-ga-in-production) | December 13, 2024 |
+| [Dynamic OS version compliance for device assurance](#dynamic-os-version-compliance-for-device-assurance) | June 14, 2023 |
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md:17]: Just want to confirm these are supposed to be 2023

File: packages/@okta/vuepress-site/docs/release-notes/2024/index.md [hunk 2]
```diff
@@ -4,6 +4,59 @@ title: Okta API Products release notes 2024
...
 > Help us improve our release notes by filling out this short [survey](https://surveys.okta.com/jfe/form/SV_4VEZcIGOX0TBgkC).
 
+## February
+
+### Monthly release 2024.02.0
+
+| Change | Expected in Preview Orgs |
+|--------|--------------------------|
+| [Assign admin roles to an app](#assign-admin-roles-to-an-app)| June 14, 2023 |
+| [DPoP support for Okta management APIs is GA in Production](#dpop-support-for-okta-management-apis-is-ga-in-production) | December 13, 2024 |
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2024/index.md:14]: just confirming this is supposed to be Dec 2024

**Review by grahamsmith-okta — APPROVED** [review]
Two questions but otherwise approved

File: packages/@okta/vuepress-site/docs/release-notes/2024/index.md (hunk 2 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2024/index.md:14]: yes

**Pull request merged**